e2e/testsuite/relayer.go

```diff
@@ -18,7 +18,7 @@ const (
 
 // newCosmosRelayer returns an instance of the go relayer.
 func newCosmosRelayer(t *testing.T, tc testconfig.TestConfig, logger *zap.Logger, dockerClient *dockerclient.Client, network string) ibc.Relayer {
-	return ibctest.NewBuiltinRelayerFactory(ibc.CosmosRly, logger, relayer.CustomDockerImage(cosmosRelayerRepository, tc.RlyTag), relayer.StartupFlags("-p", "events")).Build(
+	return ibctest.NewBuiltinRelayerFactory(ibc.CosmosRly, logger, relayer.CustomDockerImage(cosmosRelayerRepository, tc.RlyTag, ""), relayer.StartupFlags("-p", "events")).Build(
```

Comment:
Unsure if this is causing the issue I'm currently seeing. The new argument is the uid and the gid for the relayer docker image. If you pass in an empty string it uses root by default, which shouldn't cause any issues. I also don't see why it would cause a "config already exists" issue

We may want to pass in a stricter uid/guid but doesn't seem support important right now